README.md

```diff
@@ -39,7 +39,7 @@ var el = {
 ### Converting Javascript Values into Elements
 
 ```javascript
-var minim = require('minim');
+var minim = require('minim').init();
```

Comment:
I would love to see this change go away. One way to do that would be to expose a default registry like we do today that just calls `init` internally. We should think about how to make this as simple as possible and I think having the minim module be a default registry instance might be an interesting idea. I actually do [that with Fury](https://github.com/apiaryio/fury.js/blob/master/src/fury.es6#L99-L112).